Dockerfile

```diff
@@ -3,6 +3,6 @@
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY manager .
-USER nonroot:nonroot
+USER 65532:65532
```

Comment:
这个数是随便写的吗？咱们有没有统一的规范

Reply:
This user id is from https://github.com/kubernetes-sigs/kubebuilder/pull/1635 which is the part of the kubebuilder v3.

Reply:
但是我看harbor这边用的都是10000
是不是我们应该统一

Reply:
This is the user id to run the harbor-operator itself, not the user id to run the harbor. 

I think we should keep it the same with kubebuilder because these files may be generated by the kubebuilder.

Reply:
I cannot read what @bitsf wrote,
But I agree with @heww : we should keep the `nonroot` user:
https://github.com/GoogleContainerTools/distroless/blob/master/examples/nonroot/BUILD#L17

Why using it here?
IMO The change should only be be applied to files under `config/`.

Reply:
This PR lets the harbor operator can run in the k8s cluster with PSP enabled.

The `config/manager/manager.yaml` was changed with `runAsUser: 65532`

Reply:
I got it, but the dockerfile does not require any changes.
The user id should be changed on runtime, not at buildtime.

The user `65532` does not exist in `gcr.io/distroless/static:nonroot` image.